### PR TITLE
Feature/509/demo all gen3

### LINKED ISSUE #509: [FEATURE] Migrate "demo_all" to gen3
**Is your feature request related to a problem? Please describe.**
Currently, `demo_all` use case is based on legacy gen2 use cases. It shall be updated to deploy the latest gen3 use cases.

### PR DESCRIPTION
**Pull request by tobigremmer-dt**
Resolves #509

### CODE REVIEW

**Comment by tobigremmer-dt** [comment]
`demo-monaco-gitops` is excluded for now, as it depends on #468

File: user-skel/ansible/roles/demo-all/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# demo-all (gen3)
+
+### Required extra vars
+
+|Variable name|Description|
+|---|---|
+|dt_environment_url_gen3|Dynatrace Gen3 environment url, e.g. `https://<YOUR ENVIRONMENT ID>.sprint.apps.dynatracelabs.com`|
```
Comment (eemrdog) [inline user-skel/ansible/roles/demo-all/README.md:7]: environment could be dev or live as well, not only sprint. 
In the explanation, "sprint" could be <YOUR ENVIRONMENT STAGE> i.e. sprint,live,dev, etc

**Review by eemrdog — APPROVED** [review]
it looks good! 
just a comment to make the env example more generic for dev,live as well but not a big deal

File: user-skel/ansible/roles/demo-all/README.md (hunk 1 above)
Comment (tobigremmer-dt) [reply user-skel/ansible/roles/demo-all/README.md:7]: Was taking a look. "e.g." means it's an example. As we're mainly using Sprint and (if I remember correctly) prod environment URLs look a bit different anyways, I leave it as is to not complicate things.

(By the way, all other use cases use the same wording)

**Pull request merged**